sprocs/instance_questions_update_score.sql

```diff
@@ -1,4 +1,5 @@
 DROP FUNCTION IF EXISTS instance_questions_update_score(bigint,bigint,bigint,bigint,text,integer,text,double precision,double precision,jsonb,bigint);
+DROP FUNCTION IF EXISTS instance_questions_update_score(bigint,bigint,bigint,bigint,text,integer,text,double precision,double precision,jsonb,jsonb,bigint);
```

Comment:
Sprocs should no longer have `DROP FUNCTION` at the top, and they should not use `CREATE OR REPLACE FUNCTION` but instead should only use `CREATE FUNCTION`.

Merging this PR with current master should fix these issues or show conflicts for them.